build/docker/docker-compose.yml

```diff
@@ -12,12 +12,13 @@ services:
       DB_PASSWORD: $DB_PASSWORD
       WAIT_FOR: database.$PROJECT_NAME_SHORT.local:5432
       ON_ERROR_STOP: "yes"
-      DOS_DB_NAME: $DOS_DB_NAME
-      DOS_DB_PASSWORD: $DOS_DB_PASSWORD
+      DB_DOS_NAME: $DB_DOS_NAME
+      DB_DOS_USERNAME: $DB_DOS_USERNAME
+      DB_DOS_PASSWORD: $DB_DOS_PASSWORD
     command:
       [
         "scripts",
-        "--run-as-connection-details-prefix=DOS_",
+        "--run-as-connection-details-prefix=DB_DOS",
```

Comment:
Well spotted Matt. However, that has changed in the automation scripts, so that underscore is part of the matching string on the other end. See the following line of code:

https://github.com/nhsd-exeter/make-devops/blob/9f2164850d735fddafb085f3b960bf4bcb3664ff/build/automation/lib/docker/image/postgres/assets/sbin/run.sh#L35

Reply:
Cool happy with that, makes more sense to add the connecting underscore when using the prefix. Allows for more flexibility when using the prefix.